### PR TITLE
Removed dependency which is not valid with package installation

### PR DESCRIPTION
**Pull request by Mike-Petersen**
Sorry, i missed to commit the require for the package installation to work.
Is that require in any way necessary for the archive?
Otherwise i would just remove it.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -104,10 +104,9 @@
   }
 
   file { '/opt/kafka/config':
-    ensure  => directory,
-    owner   => 'kafka',
-    group   => 'kafka',
-    require => Archive["${package_dir}/${basefilename}"],
```
Comment (igalic) [inline manifests/init.pp]: perhaps we should put this dependency elsewhere then?

Comment (Mike-Petersen) [inline manifests/init.pp]: I now added the dependency to the archive/package

**Comment by Mike-Petersen** [comment]
I have retriggered the build cause it was an bundle error but it seems that there is now a problem with the new puppet 4.6 version (10.08.16). This error is not due to the code.

The problem is:
if versioncmp($kafka::version, '0.9.0.0') < 0 {

which causes an "Unknown variable: 'kafka::version'" error since puppet 4.6

Does anyone know how to fix this?

**Comment by bastelfreak** [comment]
I noticed the same yesterday, a lot of stuff is currently breaking because of puppet4. Can you please pin puppet to ~> 4.5.0 in the Gemfile? Then we can see if the tests are working.

**Comment by Mike-Petersen** [comment]
I also updated the .travis.yml to use puppet ~> 4.5.0

**Comment by fessyfoo** [comment]
this totally makes sense.

**Pull request merged**